packages/@pollyjs/adapter-puppeteer/src/index.js

```diff
@@ -220,6 +210,17 @@ export default class PuppeteerAdapter extends Adapter {
     }
   }
 
+  respondToRequest(pollyRequest) {
```

Comment:
This public hook was introduced in the base adapter refactoring so just moving the related logic here.